packages/block-library/src/text-columns/index.js

```diff
@@ -18,44 +18,18 @@ import {
 } from '@wordpress/block-editor';
 import deprecated from '@wordpress/deprecated';
 
-export const name = 'core/text-columns';
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
```

Comment:
It gets transpiled into:

![Screen Shot 2019-03-21 at 16 47 22](https://user-images.githubusercontent.com/699132/54765008-0be4e180-4bf9-11e9-90e9-2a562130ed64.png)